local_infrastructure/docker-compose.yml

```diff
@@ -54,6 +54,23 @@ services:
       - ./mysql57_databases:/var/lib/mysql
       - ./my.cnf:/etc/my.cnf
 
+  mariadb103:
+    container_name: mariadb103
+    image: 'bitnami/mariadb:10.3'
+    user: root
+    restart: always
+    labels:
+      - 'traefik.enable=false'
```

Comment:
@Almightily Please. change this label to be without quotations marks. We'll also change other labels in this file